docs/getting-started/dependencies-for-some-community-packages.md

```diff
@@ -61,32 +68,56 @@ The instructions below have _nothing to do with_ **Visual Studio Code**. If you
 
 You **do not** need a full installation of Visual Studio to be able to compile native Node modules. But you do need some components that must be installed via the Visual Studio installer, so [visit this page](https://visualstudio.microsoft.com/downloads/) and search for **Build Tools for Visual Studio 2022**. You will download an installer pre-configured to install these components.
 
-:::tip
+:::info
+
 If you want to install the Visual Studio IDE — or already have it installed — then you can use that _instead of_ Build Tools for Visual Studio, as long as you still install the **Desktop development with C++** component as described below.
 
-If you have a pre-existing installation of Visual Studio 2019, or its build tools, then those will also work just fine.
+If you have a pre-existing installation of Visual Studio 2019, or its build tools, that’ll work fine — though you might have to re-run the Visual Studio installer in order to add the **Desktop development with C++** component.
+
 :::
 
-You must also select the **Desktop development with C++** component when customizing your installation. These are the components that will allow Node to compile native modules with the `node-gyp` tool.
+You _must_ also select the **Desktop development with C++** component when customizing your installation. These are the components that will allow Node to compile native modules with the `node-gyp` tool.
 
 ![selecting the C++ component](/img/atom/vs-cpp-selected-inset.png)
 
-You will not need anything else.
+:::tip
```

Comment:
I wasn't aware of this one, from my understanding the "Windows 10 SDK" is only needed to build native modules in a community package if the user is not on Windows 10.

I may be wrong on this, but just wanting to double check.

Reply:
I'm on Windows 10 myself. I was able to install `x-terminal-reloaded` with only the C++ development tools, but I wasn't able to run `yarn install` until I installed the Windows SDK.

Reply:
Interesting, I've had it installed from past projects, so must've not realized it being a requirement. Thanks for the info!